### PR TITLE
Update scrapyapperyio.py

### PR DESCRIPTION
**Pull request by dragonabyss**
Encode response.body with utf-8, otherwise the json.loads function will fail.

### CODE REVIEW

**Comment by lookfwd** [comment]
Thanks @dragonabyss! Will have a look at it in the next 2 days!

File: scrapyapperyio.py [hunk 1]
```diff
@@ -55,7 +58,7 @@ def process_item(self, item, spider):
             def extract_session(response):
                 if response.status != 200:
                     raise RuntimeError("Unable to login: %s" % response.body)
-                self._session = json.loads(response.body)['sessionToken']
+                self._session = json.loads(str(response.body,encoding='utf-8'))['sessionToken']
```
Comment (lookfwd) [inline scrapyapperyio.py]: It all looks perfect - I would add a space after the ',' for pep8'ish (although the rest of the file isn't right now)

Comment (dragonabyss) [reply scrapyapperyio.py]: Yes，and I have added a space after comma.

**Comment by lookfwd** [comment]
Thanks a lot! :)

**Pull request merged**